zer0-os/zOS
# feat: refit message three dot menu and improvements

### What does this do?
- Refits the message hover state producing the “three dot” menu to display outside of the message and not to overlay it (as it currently does).

### Why are we making this change?
- as per design and to improve the UX

### How do I test this?
- open a chat (full messenger or collapsed) and hover over a message sent by owner or non owner. Three dots menu should transition into view.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

https://github.com/zer0-os/zOS/assets/39112648/072be051-bf15-4976-9c32-235887ec3347
